main.py

```diff
@@ -76,118 +84,52 @@ async def on_ready(self):
         if not self.synced:
             await tree.sync(guild=discord.Object(id=aloraID))
             self.synced = True
-        logger.warning(f"Debug mode is on, remember about it")
+
+        # try:
+        #     cursor.execute("CREATE")
+        # except Error as e:
+        #     logger.error(f"{e}")
+        #     return None
```

Comment:
Don't leave the commented code, if you need some snippets to test, just create the snippet in your IDE:
https://code.visualstudio.com/docs/editor/userdefinedsnippets
https://www.jetbrains.com/help/pycharm/tutorial-creating-and-applying-live-templates-code-snippets.html